### PR TITLE
couple of fixes

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Fix citizens thinking they are at the hospital when they're not
- Reduce miner pull-ahead to 4, (10 is a bit much)
- Fix courier picking up stuff from builder (this will probably also improve builder pickup efficiency)
- Fix exceptions when firing couriers with active tasks
- Already add better DO placement handling

[ x ] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

**Review by marchermans — DISMISSED** [review]

**Review by Thodor12 — DISMISSED** [review]

File: src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingMiner.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public int getResourceBatchMultiplier()
         if (getModuleMatching(WorkerBuildingModule.class, m -> m.getJobEntry() == ModJobs.quarrier.get()).getAssignedCitizen().isEmpty())
         {
             //Ask for 10x the resources if possible
-            return 10;
```
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingMiner.java:125]: It looks like the comment needs to be changed too

File: src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ public void tick()
             final int citizenModifier = citizen.getCitizenJobHandler().getColonyJob() == null ? 1 : citizen.getCitizenJobHandler().getColonyJob().getDiseaseModifier();
             final int configModifier = MineColonies.getConfig().getServer().diseaseModifier.get();
             if (!IColonyManager.getInstance().getCompatibilityManager().getDiseases().isEmpty() &&
-                  citizen.getRandom().nextInt(configModifier * DISEASE_FACTOR) < citizenModifier)
+                  citizen.getRandom().nextInt(configModifier * DISEASE_FACTOR/1) < citizenModifier)
```
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java]: Curious what the division by 1 does

Comment (Thodor12) [reply src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java]: Probably needed for type changing from int to a double or something

Comment (someaddons) [reply src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java]: bug, fix it! :D

Comment (Raycoms) [reply src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java]: debug value derp, fixed

**Review by someaddons — APPROVED** [review]

**Pull request merged**